### DISCUSSION TITLE
Limit number of connected clients to server

### DISCUSSION THREAD

**Discussion by mvelayati** [jchristn/SuperSimpleTcp#135] [Q&A]
Is there a way to limit the total number of clients allowed to connect to server?

**Comment by jchristn** [comment]
Hi @mvelayati there is not.  I'd be happy to take a PR on this if you're interested in contributing!

**Comment by mvelayati** [comment]
Hi @jchristn,
Yeah I'd be happy to contribute!

**Reply by jchristn** [reply]
Thanks @mvelayati - just added to NuGet as v2.6.1 - https://www.nuget.org/packages/SuperSimpleTcp/2.6.1

Commit: https://github.com/jchristn/SuperSimpleTcp/commit/9919bc5203551c9a85ef5a211879b57a759f3c4f
